bittensor/utils/networking.py

```diff
@@ -191,7 +191,7 @@ def wrapper(self, *args, **kwargs):
             )
             != 0
         ):
-            logging.info("Reconnection substrate...")
+            logging.debug("Reconnection substrate...")
```

Comment:
Since you're already changing this line, fix the English. 

```suggestion
            logging.debug("Reconnecting to  substrate...")
```